### ISSUE TITLE
Wrong lastest update 1.3.22

### ISSUE THREAD

**Issue by giautm** [prisma-labs/graphql-playground#476]
I download version 1.3.23 from GitHub release pages.
After install, the app says has a new update and preparing. Then, I jump to 1.3.22 again. WTF.
EDITED: I using `graphql-playground-electron-1.3.23-mac.zip`
#### This issue pertains to the following package(s):

- [x] GraphQL Playground - Electron App
- [ ] GraphQL Playground HTML
- [ ] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

#### What steps may we take to reproduce the behavior?

_Please provide a gif or image of the issue for a quicker response/fix._

**Comment by giautm** [comment]
I think update API should return largest version instead of lastest of beta.

```
https://hazel-server-kygnjrcroc.now.sh/update/mac/1.3.23
```
Response is
```
{
"name": "v1.3.22",
"notes": "## Fixes:\r\n\r\n- Variable editor is broken #372\r\n- Copying curl doesn't work for custom header Authorization #333",
"pub_date": "2018-01-05T20:45:48Z",
"url": "https://github.com/graphcool/graphql-playground/releases/download/v1.3.22/graphql-playground-electron-1.3.22-mac.zip"
}
```

**Comment by Cervajz** [comment]
Just confirming that this happens to me as well. 

Steps to reproduce:

1) Download the latest stable version (1.4.4 in my case)
2) Set `betaUpdates` in the settings to `true`
3) Restart the app and check for updates
4) The latest beta version is downloaded and replace the latest stable version
